@ndlib/gatsby-theme-marble/src/components/Shared/SearchTools/SearchRefinementListFilter/index.js

```diff
@@ -4,15 +4,17 @@ import {
   RefinementListFilter,
 } from 'searchkit'
 
-const listOrder = (list) => {
-  list.sort((a, b) => {
-    return a.key.localeCompare(b.key, undefined, { numeric: true })
-  })
+const listOrder = (list, sort) => {
+  if (sort === 'a-z') {
+    list.sort((a, b) => {
+      return a.key.localeCompare(b.key, undefined, { numeric: true })
```

Comment:
@jonhartzler @jmille79 Do we need a check here that all items in the list have a valid key? Given some of the assumptions we've made on data from elastic search before it seems like we should do a check.

Maybe:
```
return a.key && b.key ? a.key.localeCompare(b.key, undefined, { numeric: true }) : 0
```

Reply:
I am actually not sure where key is coming form.  Key is not something that I am setting somewhere in the chain.  So it could be ok but I do think we should error on a more careful run a do the safer check.